ASM/src/build.asm

```diff
@@ -142,6 +140,8 @@ TRIFORCE_ICON_TEXTURE:
 
 .align 0x10
 
+.skip 0x200 ; Temporary address bump to avoid audio issues
```

Comment:
![A Minecraft screenshot showing a simple bridge crossing a narrow river. In front of the bridge, there are signs reading “Temporary Bridge (don't judge)”, “Temporary means Permanent (not judging)”, and “h*ck”](https://github.com/OoTRandomizer/OoT-Randomizer/assets/641386/b7b08747-8b24-42cf-b7f5-b9dec83023f2)